src/main/java/com/ibm/devops/connect/CRPipeline/UploadJUnitTestResult.java

```diff
@@ -149,9 +149,9 @@ public FileUploader(Map<String, String> properties, TaskListener listener, Strin
                 options.put("combineTestSuites", combineTestSuites.toString());
             }
 
-            payload.put("metricName", testSetName);
+            payload.put("dataSet", testSetName);
```

Comment:
why is this changing? did it change on the api side?

Reply:
About 3 months ago we went through a large renaming  effort. After that time both were supported and internally we renamed the old to the new.  This was a good oppotuniy to use the newer name.